eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/TwoNodesIntegrationSpec.scala

```diff
@@ -0,0 +1,96 @@
+package fr.acinq.eclair.integration.basic
+
+import fr.acinq.bitcoin.scalacompat.{ByteVector32, SatoshiLong}
+import fr.acinq.eclair.channel.{DATA_NORMAL, NORMAL}
+import fr.acinq.eclair.integration.basic.fixtures.TwoNodesFixture
+import fr.acinq.eclair.testutils.FixtureSpec
+import fr.acinq.eclair.{BlockHeight, MilliSatoshiLong}
+import org.scalatest.TestData
+import org.scalatest.concurrent.IntegrationPatience
+import scodec.bits.HexStringSyntax
+
+
+/**
+ * This test checks the integration between Channel and Router (events, etc.)
+ */
+class TwoNodesIntegrationSpec extends FixtureSpec with IntegrationPatience {
+
+  type FixtureParam = TwoNodesFixture
+
+  import fr.acinq.eclair.integration.basic.fixtures.MinimalNodeFixture._
+
+  override def createFixture(testData: TestData): FixtureParam = {
+    // seeds have been chose so that node ids start with 02aaaa for alice, 02bbbb for bob, etc.
```

Comment:
nit:
```suggestion
    // seeds have been chosen so that node ids start with 02aaaa for alice, 02bbbb for bob, etc.
```

Reply:
Fixed in 3d42e3af8.

Reply:
A nice side effect of this trick is that node ids are now ordered, which means that in channel alice-bob, update_1 will always be for alice, and update_2 for bob. That is *really* convenient for debugging the router.